plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+        type: dict
+        contains:
+            status:
+                description:
+                - The current status of the global cluster. Possible values are as follows
+                type: str
+                sample: "pending"
+            from_db_cluster_arn:
+                description: The Amazon Resource Name (ARN) of the Aurora DB cluster that is currently being demoted, and which is associated with this state.
+                type: str
+                sample: arn:aws:rds:us-east-1:123456789012:cluster:ansible-test-primary
+            to_db_cluster_arn:
+                description: The Amazon Resource Name (ARN) of the Aurora DB cluster that is currently being promoted, and which is associated with this state.
+                type: str
+                sample: arn:aws:rds:us-east-2:123456789012:cluster:ansible-test-secondary
+            is_data_loss_allowed:
+                description:
+                - Indicates whether the operation is a global switchover or a global failover.
+                - If data loss is allowed, then the operation is a global failover. Otherwise, it is a switchover.
+                type: bool
+                sample: false
+"""
+
+
+try:
+    import botocore
+except ImportError:
+    pass  # handled by AnsibleAWSModule
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+
+from ansible_collections.amazon.aws.plugins.module_utils.botocore import is_boto3_error_code
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
+from ansible_collections.amazon.aws.plugins.module_utils.transformation import ansible_dict_to_boto3_filter_list
+
+
+@AWSRetry.jittered_backoff(retries=10)
+def _describe_global_clusters(client, **params):
+    try:
+        paginator = client.get_paginator("describe_global_clusters")
+        return paginator.paginate(**params).build_full_result()["GlobalClusters"]
+    except is_boto3_error_code("GlobalClusterNotFoundFault"):
+        return []
+
+
+def cluster_info(client, module):
+    global_cluster_id = module.params.get("global_cluster_identifier")
+    filters = module.params.get("filters")
+
+    params = dict()
+    if global_cluster_id:
+        params["GlobalClusterIdentifier"] = global_cluster_id
+    if filters:
+        params["Filters"] = ansible_dict_to_boto3_filter_list(filters)
+
+    try:
+        result = _describe_global_clusters(client, **params)
+    except (botocore.exceptions.ClientError, botocore.exceptions.BotoCoreError) as e:
+        module.fail_json_aws(e, "Couldn't get Global cluster information.")
+
+    return dict(
+        changed=False, global_clusters=[camel_dict_to_snake_dict(cluster, ignore_list=["Tags"]) for cluster in result]
+    )
+
+
+def main():
+    argument_spec = dict(
+        global_cluster_identifier=dict(),
```

Comment:
I guess you can specify the type here.

Reply:
Since the default value for "type" is "str," it can be left unpopulated.